README.md

```diff
@@ -313,3 +313,18 @@ This can by compiled using a command like the following after running
 `make install` on VkRunner:
 
     cc -o myrunner myrunner.c $(pkg-config --cflags --libs vkrunner)
+
+## Android
+
+VkRunner supports Android NDK build which generates the VkRunner shared library
+for Android.
+
+- Download [Android NDK](https://developer.android.com/ndk/downloads/).
+- Run the following command.
+
+```
+$ cd android_ndk/jni
+$ path/to/ndk/ndk-build
```

Comment:
These build instructions don’t seem to work without a prior build of a non-Android version because nothing generates the config.h.

Reply:
Thanks. Added the cmake build command to the description.